h2o-core/src/main/java/hex/ModelMetricsRegression.java

```diff
@@ -158,8 +168,10 @@ public ModelMetrics makeModelMetrics(Model m, Frame f, Frame adaptedFrame, Frame
 
           meanResDeviance = new MeanResidualDeviance(_dist, preds.anyVec(), actual, weight).exec().meanResidualDeviance;
         }
-      } else {
+      } else if((m != null && m._parms._distribution != DistributionFamily.custom) || (_dist != null && _dist.distribution != DistributionFamily.custom) ) {
```

Comment:
can you please explain why can we not really on a single source of the family parameter?

Reply:
Because `Model m` can be null and then it throws null pointer exception.